### DISCUSSION TITLE
[Dreambooth Lora] Is it normal for the 5.2 Dataset Configuration part to take a while?

### DISCUSSION THREAD

**Discussion by jatazgo** [Linaqruf/kohya-trainer#214] [Q&A] [answered]
Does it depend on how many images that are in the dataset? I have about 65 images and it takes the full 25 system ram of the colab.

**Comment by jatazgo** [comment, marked as answer]
Well I left reg_data_dir empty so it caused some kind of infinite recursion loop or something.

**Reply by chepy21** [reply]
I run into the same problem. I do not have a set of Regularization images. The notebook said it's optional so I left it blank. How did you solve your problem? Is this notebook not working if there's no Regularization images? thanks!
